sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -554,6 +558,10 @@ rowFormat
       (NULL DEFINED AS nullDefinedAs=STRING)?                                       #rowFormatDelimited
     ;
 
+multiPartIdentifier
```

Comment:
Any name better than `multiPartIdentifier`?

Reply:
I think this is good. It is descriptive.